gini/gini-vision-lib-ios
# Remove support for iOS < 9.0

Removed support for iOS < 9.0.

### How to test
Run `pod install` and see that everything is created successfully and the projects builds.

### Merging
Automatic
